modules/julia/gen/gen3_cpp.py

```diff
@@ -0,0 +1,301 @@
+
+#!/usr/bin/env python3
```

Comment:
Shebang should go to the first line.

Add short license header after that (with `#` comments)

Reply:
Should I add the license to all python files?

Reply:
It would be nice to update .py files from this PR.

Reply:
Is it fine now?